dipy/reconst/tests/test_ivim.py

```diff
@@ -165,10 +165,10 @@ def test_bounds_x0():
     from dipy.data.fetcher. These are values from the voxel [160, 98, 33]
     which can be obtained by :
     
-    >>> from dipy.data.fetcher import read_ivim
-    >>> img, gtab = read_ivim()
-    >>> data = img.get_data()
-    >>> signal = data[160, 98, 33, :]
+    # from dipy.data.fetcher import read_ivim
+    # img, gtab = read_ivim()
+    # data = img.get_data()
+    # signal = data[160, 98, 33, :]
```

Comment:
Yeah - I didn't think about that. I think that the best way to insert this here is with the sphinx code-block directive: http://www.sphinx-doc.org/en/stable/markup/code.html#directive-code-block